tests/test_cacher.py

```diff
@@ -116,11 +117,37 @@ async def func(*args, **kwargs):
     monkeypatch.setattr(AlmaTestsCacher, 'bulk_create_test_folders', func)
 
 
+@pytest.fixture
+def mock_clone_git_repo(
+        monkeypatch: pytest.MonkeyPatch,
+        tmp_path: Path,
+):
+    def func(*args, **kwargs):
+        test_dir = tmp_path / "test_dir"
+        test_dir.mkdir(parents=True, exist_ok=True)
+
+        sub_dir1 = test_dir / "subdir1"
+        sub_dir2 = test_dir / "subdir2"
+        sub_dir1.mkdir()
+        sub_dir2.mkdir()
+
+        file1 = test_dir / "file1.txt"
+        file2 = sub_dir1 / "file2.txt"
+        file1.write_text("This is file1.")
+        file2.write_text("This is file2.")
+
+        file3 = sub_dir2 / "file3.txt"
+        file3.write_text("This is file3.")
```

Comment:
actually, there is no need to create files, you just need to create dirs that can be found by `tests_prefix` from `repo_payload` fixture